includes/class-newspack-popups-segmentation.php

```diff
@@ -386,6 +408,68 @@ function( $segment ) {
 		return $segments;
 	}
 
+	/**
+	 * Get array of default segments by type.
+	 *
+	 * @return array Array of default segments.
+	 */
+	public static function get_default_segments() {
+		return get_option( self::DEFAULT_SEGMENTS_OPTION_NAME, false );
+	}
+
+	/**
+	 * Generate default segments. If they've already been generated, rebuild them with default parameters.
+	 *
+	 * @return array Array of all segments.
+	 */
+	public static function create_default_segments() {
+		$default_segments = get_option( self::DEFAULT_SEGMENTS_OPTION_NAME, [] );
+		$default_options  = [
+			'donors'           => [
+				'name'          => __( ' Donors', 'newspack-popups' ),
+				'configuration' => [
+					'is_donor' => true,
+				],
+			],
+			'frequent_readers' => [
+				'name'          => __( ' Frequent Readers', 'newspack-popups' ),
+				'configuration' => [
+					'min_posts' => 5,
+				],
+			],
+			'subscribers'      => [
+				'name'          => __( ' Newsletter Subscribers', 'newspack-popups' ),
+				'configuration' => [
+					'is_subscribed' => true,
+				],
+			],
+			'everyone_else'    => [
+				'name'          => __( ' Other Readers', 'newspack-popups' ),
```

Comment:
Nit: there are extra leading spaces in the segments names:
```suggestion
				'name'          => __( 'Other Readers', 'newspack-popups' ),
```